### PR TITLE
Set up SMS to Download experiment

### PR DESCRIPTION
**Pull request by robhudson**
## Issue / Bugzilla link

#11119 

## Testing

Email: http://localhost:8000/firefox/browsers/mobile/ios/?v=1
SMS: http://localhost:8000/firefox/browsers/mobile/ios/?v=2
QR: http://localhost:8000/firefox/browsers/mobile/ios/?v=3

### CODE REVIEW

File: bedrock/firefox/tests/test_views.py [hunk 1]
```diff
@@ -641,3 +639,74 @@ def test_firefox_welcome_page1(self, render_mock):
...
+            self.url,
+            data=params,
+        )
+
+    @patch("bedrock.firefox.views.TWILIO_CLIENT")
+    def test_no_data(self, mock_client):
+        resp = self._do_post({})
+        assert resp.status_code == 200
+        data = resp.json()
+        assert data["success"] is False
```
Comment (pmac) [inline bedrock/firefox/tests/test_views.py:664]: nit: better to just use `assert not data["success"]`? Don't normally use the `is` keyword with booleans, but I guess it is more specific. Either way.

**Review by pmac — APPROVED** [review]
Looks good to me. If the front-end bits have been reviewed and approved then let's go ➡️

File: bedrock/firefox/tests/test_views.py (hunk 1 above)
Comment (robhudson) [reply bedrock/firefox/tests/test_views.py:664]: Makes sense. This follows the pattern already in use in this file in other tests. But I agree simply `if cond:` or `if not cond:` is more Pythonic.

File: bedrock/firefox/templates/firefox/browsers/mobile/includes/s2d-ios-sms.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+<div class="c-inline-form">
+  <h2 class="c-inline-form-title">{{ ftl('mobile-ios-get-firefox-for') }}</h2>
+  <p class="c-inline-form-body">{{ ftl('mobile-ios-send-a-download-link-to-your') }}</p>
+  <section id="s2d-sms" class="send-to-device vertical" data-spinner-color="#000" data-countries="US">
+    <div class="form-container">
+      <h2 class="mzp-u-title-xs thank-you hidden">{{ ftl('send-to-device-your-download-link') }}</h2>
+      <form class="send-to-device-form" action="{{ url('firefox.sms-send-to-device-post') }}" method="post">
+        <ul class="mzp-c-form-errors hidden">
+          <li class="number">Enter a 10-digit phone number</li>
+          <li class="sms">Sorry, we can’t send SMS messages to this phone number.</li>
```
Comment (nathan-barrett) [inline bedrock/firefox/templates/firefox/browsers/mobile/includes/s2d-ios-sms.html:18]: Should these strings be localized?

**Review by nathan-barrett — APPROVED** [review]
r+wc, all FE stuff looks good to me, just a question about translations

File: bedrock/firefox/templates/firefox/browsers/mobile/includes/s2d-ios-sms.html (hunk 2 above)
Comment (craigcook) [reply bedrock/firefox/templates/firefox/browsers/mobile/includes/s2d-ios-sms.html:18]: The experiment is English-only, but if this becomes permanent we'll localize stuff.

**Pull request merged**